data/json/monsters/zed_tentacle.json

```
@@ -0,0 +1,190 @@
+    "melee_dice": 2,
+    "melee_dice_sides": 3,
+    "melee_damage": [ { "damage_type": "bash", "amount": 0 } ],
+    "weakpoint_sets": [ "wps_humanoid_body" ],
+    "families": [ "prof_intro_biology", "prof_physiology", "prof_wp_zombie" ],
+    "vision_night": 3,
+    "harvest": "zombie_humanoid",
+    "grab_strength": 12,
+    "special_attacks": [
+      { "id": "grab", "cooldown": 5, "move_cost": 50 },
+      { "id": "grab_2", "cooldown": 5, "move_cost": 50 },
+      {
+        "id": "stretch_attack",
+        "cooldown": 5,
+        "move_cost": 50,
+        "range": 2,
+        "damage_max_instance": [ { "damage_type": "bash", "amount": 4 } ]
+      },
+      { "id": "tentacle", "cooldown": 15, "damage_max_instance": [ { "damage_type": "bash", "amount": 3 } ] },
+      { "id": "ranged_pull", "cooldown": 10, "grab_data": { "pull_weight_ratio": 1 } }
+    ],
+    "upgrades": { "half_life": 24, "into": "mon_zombie_strider" },
+    "flags": [
+      "SEES",
+      "HEARS",
+      "STUMBLES",
+      "WARM",
+      "GRABS",
+      "BASHES",
+      "GROUP_BASH",
+      "POISON",
+      "NO_BREATHE",
+      "REVIVES",
+      "PUSH_MON",
+      "FILTHY"
+    ],
+    "armor": { "electric": 1 }
+  },
+  {
+    "id": "mon_zombie_strider",
+    "type": "MONSTER",
+    "name": { "str": "zombie strider" },
+    "description": "In front of you lies the most disgusting rendition of a starfish you've ever seen.  Not just the arms, but the legs and even the head have been horrendously deformed and stretched several meters long.  It walks around hunchback, looking through the remains of its head for its next victim.",
+    "default_faction": "zombie",
+    "bodytype": "human",
+    "species": [ "ZOMBIE", "HUMAN" ],
+    "volume": "62500 ml",
+    "weight": "81500 g",
+    "diff": 10,
+    "hp": 180,
+    "speed": 60,
+    "material": [ "flesh" ],
+    "symbol": "Z",
+    "color": "dark_gray_cyan",
+    "scents_tracked": [ "sc_human", "sc_fetid" ],
+    "aggression": 100,
+    "morale": 100,
+    "melee_skill": 6,
+    "melee_dice": 2,
+    "melee_dice_sides": 3,
+    "melee_damage": [ { "damage_type": "bash", "amount": 0 } ],
+    "weakpoint_sets": [ "wps_humanoid_body" ],
+    "families": [ "prof_intro_biology", "prof_physiology", "prof_wp_zombie", "prof_wp_hulk" ],
+    "vision_day": 8,
+    "vision_night": 3,
+    "harvest": "zombie_humanoid",
+    "grab_strength": 30,
+    "special_attacks": [
+      { "id": "grab", "cooldown": 5, "move_cost": 50 },
+      { "id": "grab_2", "cooldown": 5, "move_cost": 50 },
+      {
+        "id": "stretch_attack",
+        "cooldown": 5,
+        "move_cost": 50,
+        "range": 2,
+        "damage_max_instance": [ { "damage_type": "bash", "amount": 4 } ]
+      },
+      { "id": "tentacle", "cooldown": 10, "damage_max_instance": [ { "damage_type": "bash", "amount": 4 } ] },
+      { "id": "ranged_pull", "cooldown": 5, "grab_data": { "pull_weight_ratio": 1.5 } },
+      {
+        "id": "teke_push",
```

```suggestion
        "id": "teke_push",
```
Probably want to change this id.  There are likely some mods that impact this special attack in it's original form that wouldn't make sense impacting this monsters ability.